### DISCUSSION TITLE
install error on ubuntu 20 using openssl 1.0.2u

### DISCUSSION THREAD

**Discussion by brunoadacosta** [rbenv/ruby-build#1849] [Build failures]
RUBY_CONFIGURE_OPTS="--with-openssl-dir=/opt/openssl-1.0.2u" ./ruby-build 2.3.8 ~/.asdf/installs/ruby/2.3.8
Downloading ruby-2.3.8.tar.bz2...
-> https://cache.ruby-lang.org/pub/ruby/2.3/ruby-2.3.8.tar.bz2
Installing ruby-2.3.8...

WARNING: ruby-2.3.8 is past its end of life and is now unsupported.
It no longer receives bug fixes or critical security updates.

ruby-build: using readline from homebrew

BUILD FAILED (Ubuntu 20.04 using ruby-build 20200520)

Inspect or clean up the working tree at /tmp/ruby-build.20200706145235.256025.tiU8z5
Results logged to /tmp/ruby-build.20200706145235.256025.log

Last 10 log lines:
The Ruby openssl extension was not compiled.
ERROR: Ruby install aborted due to missing extensions
Try running `apt-get install -y libssl-dev` to fetch missing dependencies.

Configure options used:
  --prefix=/home/bruno/.asdf/installs/ruby/2.3.8
  --with-openssl-dir=/opt/openssl-1.0.2u
  --with-readline-dir=/home/linuxbrew/.linuxbrew/opt/readline
  LDFLAGS=-L/home/bruno/.asdf/installs/ruby/2.3.8/lib 
  CPPFLAGS=-I/home/bruno/.asdf/installs/ruby/2.3.8/include

**Comment by veejar** [comment]
https://www.garron.me/en/linux/install-ruby-2-3-3-ubuntu.html
